packages/oui-field/README.md

```diff
@@ -0,0 +1,119 @@
+# oui-field
+
+<component-status cx-design="partial" ux="beta"></component-status>
+
+oui-field is a package which provides styles for form fields.
+
+## Installation
+
+```less
+  @import 'oui-form/form';
+```
+
+## Usage
+
+### Standard sizes
+
+```html:preview
+<div class="oui-field">
+  <label for="text" class="oui-field__label oui-label">Input size 1</label>
+  <div class="oui-field__content">
+    <div class="oui-field-control oui-field-control_1">
+      <input type="text" id="text" name="text" class="oui-input" value="Input text" />
+    </div>
+    <div class="oui-field-helper">An input with oui-field-control_1 class</div>
```

Comment:
Do you think it would be better if it was a label instead of a div. If the user click on this "information" text it sould focus the input, no? (Maybe the CX team would be better to answer that?)